source/docs/contracts_configuration.md

```diff
@@ -1,322 +1,327 @@
-
-<pre><code class="javascript">// config/contracts.js
-module.exports = {
-  ...
-  "development": {
-    "contracts": {
-      "SimpleStorage": {
-        "args": [
-          100,
-          <mark class="highlight-inline">"$MyStorage"</mark>
-        ]
-      },
-      "MyStorage": {
-        "args": [
-          "initial string"
-        ]
-      },
-      "MyMainContract": {
-        "args": [
-          <mark class="highlight-inline">"$SimpleStorage"</mark>
-        ]
-      },
-      "MyContractInteface": {
-        "deploy": false
-      }
-    }
+<pre><code class="javascript">...
+contracts: {
+  SimpleStorage: {
+    args: { initialValue: 100 }
   }
-  ...
 }
+...
 </code></pre>
 
-### Multiple Instances
-
-You can deploy many instances of the same contract. E.g.
-
-<pre><code class="javascript">// config/contracts.js
-module.exports = {
-  "development": {
-    "contracts": {
-      "Currency": {
-        "deploy": false,
-        "args": [
-          100
-        ]
-      },
-      "Usd": {
-        <mark class="highlight-inline">"instanceOf": "Currency",</mark>
-        "args": [
-          200
-        ]
-      },
-      "MyCoin": {
-        <mark class="highlight-inline">"instanceOf": "Currency",</mark>
-        "args": [
-          200
-        ]
-      }
+### Configuring gas and gas price
+
+Both, `gas` and `gasPrice` can be configured for each Smart Contract. If we don't want to configure that for every single contract, we can also specify `gas: auto` in the environment, like this:
+
+<pre><code class="javascript">...
+development: {
+  gas: 'auto',
+  contracts: {
+    SimpleStorage: {
+      args: [100],
+      gas: 800000,
+      gasPrice: 5
     }
   }
 }
+...
 </code></pre>
 
-### Static Addresses
-
-Contracts' addresses can be defined. If an address is defined the contract wouldn't be deployed but its defined address will be used instead.
-
-<pre><code class="javascript">// config/contracts.js
-module.exports = {
-  // ...
-  "development": {
-    "contracts": {
-      "UserStorage": {
-        <mark class="highlight-inline">"address": "0x123456"</mark>
-      },
-      "UserManagement": {
-        "args": [
-          "$UserStorage"
-        ]
-      }
+Another cool feature of Embark is that it support human readable ether units, to improve the developer experience..
+
+{% note info Human readable ether units %}
+
+Embark support **human readable ether units** in places where ether unit values are required. [Read here](#Human-readable-Ether-units) for more information.
+{% endnote %}
+
+## Configuring Smart Contract Dependencies
+
+When building more complex applications, it's very common that a Smart Contract depends on another one. Usually, that's when we need the address of the dependency Smart Contract. Embark makes it very easy to not only ensure dependency Smart Contracts are deployed before the Smart Contract in question deploys, but also accessing their deployed addresses. 
```

Comment:
```suggestion
When building more complex applications, it's very common that a Smart Contract depends on another one. Embark makes it very easy to not only ensure dependency Smart Contracts are deployed before the Smart Contract in question deploys, but also accessing their deployed addresses. 
```
Don't feel like that sentence dded much.  If you want to keep it, I would at least drop the "Smart Contract" and just keep "dependency"